src/lighthouse.js

```diff
@@ -110,8 +112,9 @@ module.exports = function(driver, opts) {
   }
 
   // Collate all artifacts required by audits to be run.
+  // userScripts is required by default
   const auditArtifacts = audits.map(audit => audit.meta.requiredArtifacts);
-  const requiredArtifacts = new Set([].concat(...auditArtifacts));
+  const requiredArtifacts = new Set(['userScripts'].concat(...auditArtifacts));
```

Comment:
maybe only add by default when there are user scripts? That way if/when we skip the initial scheduler pass if no audits need first pass artifacts or the trace, `userScripts` will be good to go as well